scripts/generate_pip_deps_from_conda.py

```diff
@@ -127,13 +127,13 @@ def main(conda_fname, pip_fname, compare=False):
     )
     if res:
         msg = (
-            "`requirements-dev.txt` has to be generated with `{}` after "
-            "`environment.yml` is modified.\n".format(sys.argv[0])
+            f"`requirements-dev.txt` has to be generated with `{sys.argv[0]}` after "
+            f"`environment.yml` is modified.\n"
```

Comment:
same

Reply:
Thanks for your feedback, I just changed it